package.json

```diff
@@ -79,13 +80,10 @@
     "typescript": "4.3.3"
   },
   "dependencies": {
-    "@mapbox/node-pre-gyp": "1.0.11",
-    "neon-cli": "0.10.1"
+    "@mapbox/node-pre-gyp": "1.0.11"
```

Comment:
QS: should this be a devDependency? (btw, I haven't used this package before, could be wrong)

Reply:
Probably, I will move it to a dev dependency, the cli as well should have been a dev dependency

Reply:
I will leave it as a normal dependency as it might be needed for distribution for other library